api/src/middlewares/services/utils/getFormType.ts

```diff
@@ -0,0 +1,12 @@
+import { AnswersHashMap } from "../../../types/AnswersHashMap";
+import { FormType } from "../../../types/FormDataBody";
+
+export function getFormType(answers: AnswersHashMap, type: FormType): FormType {
+  if (answers.service) {
+    return answers.service as FormType;
+  }
+  if (answers.over16 !== undefined) {
```

Comment:
it looks like this is used to select the right notify template. can we change `getUserTemplate` instead?